test/test_css_parser_loading.rb

```diff
@@ -132,6 +132,62 @@ def test_following_remote_import_rules
     end
   end
 
+  def test_allowing_at_import_rules_from_add_block
+    css_block = "@import 'https://fonts.googleapis.com/css?family=Suez+One';"
+
+    parser = Parser.new
+    parser.add_block!(css_block, :ignore_import => false)
+
+    if css_block.include? "@import"
+      result = "Found import"
+    end
+
+    assert_equal "Found import", result
+  end
+
+  def test_ignoring_at_import_rules_from_add_block
+    css_block = "@import 'https://fonts.googleapis.com/css?family=Suez+One';"
+
+    parser = Parser.new
+    parser.add_block!(css_block, :ignore_import => true)
+
+    unless css_block.include? "@import"
+      result = "import not found"
+    end
+
+    assert_equal "import not found", result
+  end
+
+  # in order for font-face to work we need to make sure new lines are
+  # removed after passing css_block to add_block!
+  def test_allowing_font_face_rule
+    # this will return a string with contents of font_face.css
+    css_block = File.open('test/fixtures/font_face.css', 'r') { |f| f.read }
+
+    parser = Parser.new
+    parser.add_block!(css_block, :allow_font_face => true)
+
+    unless css_block.include? "\n"
+      result = "no new lines found"
+    end
+
+    assert_equal "no new lines found", result
+  end
+
+  def test_not_allowing_font_face_rule
+    # this will return a string with contents of font_face.css
+    css_block = File.open('test/fixtures/font_face.css', 'r') { |f| f.read }
```

Comment:
Why extract this out to the file? For the test case it doesn't need to be so big (nor reference objects that actually exist), it just needs to have *a* newline and parse correctly.